gage-langdon/lovlettr
# feat: init jest, create label component

![image](https://user-images.githubusercontent.com/8952305/54487068-5ad3f500-484e-11e9-9dd7-e36863a262c5.png)


**New**
 - Initialized Jest for component snapshot testing
   - Initialized Babel with Gatsby preset in order for Jest to run es6 and jsx
 - Created label component with tests!

**Steps to verify**
 - Look at the code in the 'files changed' tab above
    -  code review - am I doing anything potential for error? Can anything be done better? 
    -  am I introducing anything malicious to the code base? 
    - you can click on the lines of code in the 'files changed' tab and leave comments or questions. You can even leave comments to ask for clarification before approving the pr. 
 - `npm checkout feat/label`
 - `npm i`
 -  `npm test` => ensure all tests pass
 -  cursive label should be on home page, ensure its rendering correctly
 -  does the feature added adhere to the original specs in the ticket and mockup?
 -  If it looks good, approve! This process should happen as fast as possible in order to not allow code to go stale.
